### ISSUE TITLE
description of the image

### ISSUE THREAD

**Issue by gonsie** [Heatherms27/BlogPost#1]
Could you clarify what the walls and water pipe image is showing? What I think it's showing is:
- 2 blue walls
- 1 blue water pipe
- the gray blob is the air in the room where the water pipe is??

We are trying to calculate the temperature of the air, correct? The text mentions that the heat equation calculates temperature of a solid... but if the gray blob is air, that's not a solid. 

I think I'm missing something 😞

**Comment by Heatherms27** [comment]
I think I'm missing something too... I was assuming that the pip touched the inside of the walls (or "boundaries"), but the image doesn't make it seem like that's the case. Also, even if the heat equation could be applied to any medium, not just a solid, the air, walls, pipe, and water all are different materials and have a different thermal diffusity. I took the image from Mark's lesson, so maybe his thought process was showing a harder, more common example of the heat equation, then simplifying it for convenience. 

I found a different example. It's not as pretty as Mark's, but it's easier to understand and applies to the 1D case. It just assumes you have a thin metal rod with some initial temperature distribution and boundary conditions at the beginning and end of the rod. Heat can only travel horizontally and there is no outside heating/cooling. The idea is the track the temperature distribution inside the rod as time progresses.

Example from http://ramanujan.math.trinity.edu/rdaileda/teach/s14/m3357/lectures/lecture_2_25_slides.pdf
